androdeb

```diff
@@ -268,6 +285,7 @@ if [ ! -z "$KERNELSRC" ]; then
 	tar -xvf $TDIR_ABS/kh.tgz -C $OUT_TMP/kernel-headers/ > /dev/null
 fi
 
+
```

Comment:
why this newline?

Reply:
Do you mean below line?
        268   285       tar -xvf $TDIR_ABS/kh.tgz -C $OUT_TMP/kernel-headers/ > /dev/n
If yes ==> This isn't new line. It used in the base code repository.
If no ==> Please tell me which line directly.